src/tests/ftest/util/bytes_utils.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2020 Intel Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+The Government's rights to use, modify, reproduce, release, perform, display,
+or disclose this software are subject to the terms of the Apache License as
+provided in Contract No. B609815.
+Any reproduction of computer software, computer software documentation, or
+portions thereof marked with this legend must also reproduce the markings.
+"""
+from logging import getLogger
+
+
+class Bytes(object):
+    """Defines a byte capacity with its units.
+
+    Allows comparison values with different units, e.g. '1.6T' to '750G'.
+    """
+
+    SIZES = ["", "K", "M", "G", "T", "P", "E", "Z", "Y"]
+
+    def __init__(self, amount, units):
+        """Initialize a Bytes object.
+
+        Args:
+            amount (int): number of bytes relative to the unit specified
+            units (str): units as a single letter
+        """
+        self._amount = 0
+        self._units = self.SIZES[0]
+
+        self.amount = amount
+        self.units = units
+
+    @property
+    def amount(self):
+        """Get the bytes amount."""
+        return self._amount
+
+    @property
+    def units(self):
+        """Get the units amount."""
+        return self._units
+
+    @amount.setter
+    def amount(self, value):
+        """Set the bytes amount.
+
+        Args:
+            value (object): number of bytes relative to the unit specified
+
+        Raises:
+            ValueError: if the value is invalid (not a number)
+
+        """
+        self.amount = float(value)
+        if str(self.amount).endswith(".0"):
+            self.amount = int(self.amount)
```

Comment:
(pylint-redefined-variable-type) Redefinition of self.amount type from float to int